setup.py

```diff
@@ -3,68 +3,74 @@
 from sys import version_info
 
 
-install_requires = [
-    'six',
-    'content-io >= 1.2.5',
-    'simplejson >= 3.2.0'
-]
+install_requires = ["six", "content-io >= 1.2.5", "simplejson >= 3.2.0"]
 
 tests_require = [
-    'coverage',
-    'Markdown <= 2.4.1',
-    'Pillow <= 3.4.2',
+    "coverage",
+    "Markdown <= 3.3.0",
+    "Pillow <= 3.4.2",
```

Comment:
We should bump this. We use `8.4.0` on bolist. Maybe at least 6?

Reply:
Why not the same?

Reply:
Actually, I don't think we should require a specific version at all